### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by bidulemachin**
minor update on Raspberry Pi (Raspbian Jessie)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,7 +112,7 @@ Do NOT run the node as root.
 The following lines show `[sudo]` as optional. Adjust accordingly. 
 
 ```bash
-[sudo] echo "deb http://ftp.debian.org/debian jessie-backports main" | tee -a /etc/apt/sources.list
+[sudo] echo "deb http://ftp.debian.org/debian jessie-backports main" | [sudo] tee -a /etc/apt/sources.list
```
Comment (jleni) [inline README.md]: I am not sure this change is correct. The first command does not need sudo, it is the second (tee) that accesses a restricted file. We should probably remove the first sudo.

File: README.md [hunk 2]
```diff
@@ -128,6 +128,19 @@ sudo pip3 install -r requirements.txt
...
+
+```dependencies not satisfied, run [pip3 install -r requirements.txt] first.
+ The 'cffi>=1.7' distribution was not found and is required by cryptography
+ get an error with 
+``` 
+
+Add the cffi package:
+
+```
+[sudo] pip3 install cffi
```
Comment (jleni) [inline README.md:141]: If this is the case, it would make more sense to add cffi to the requirements.txt file

**Comment by jleni** [comment]
Hi @bidulemachin Thanks for the PR! I will have a look

File: README.md (hunk 1 above)
Comment (bidulemachin) [reply README.md]: I think you are right. There is no [sudo] on Debian Jessie instructions.

File: README.md (hunk 2 above)
Comment (bidulemachin) [reply README.md:141]: I made this suggestion to Cyyber too.
It look like it is not required on newer Linux version.

**Comment by jleni** [comment]
Ok, I will merge. This is all because of the latest version of twisted imposed all the new requirements. In a few weeks when twisted is completely out we can revise these instructions.

**Comment by jleni** [comment]
Thanks @bidulemachin !

**Pull request merged**